prow/private_deck.md

```diff
@@ -0,0 +1,166 @@
+# Setting up Private Deck
+
+## 1) Create a PR to Set up TenantIDs for ProwJobs and Repos
+
+Prow users should create a PR creating tenantID defaults for their org/repos and clusters. Once you set up a tenantID, all prowjobs labelled with that tenantID will only be visible on Deck instances created with the same tenantID. If you already have prowjobs that you don't want to lose access to on Deck, do this step last. If not, do it first to make sure prowjobs you want to keep sequestered do not appear on other instances of Deck.
+
+The recommended way to add TenantIDs to Prowjobs based on org/repo or cluster is through prowjob_default_entries in the prow config. This will apply the tenant ID to jobs with matching cluster AND repo. If you want to do cluster OR repo, create two entries in the config and use "*" for either field.
+```yaml
+prowjob_default_entries:
+  - cluster: "build-private"
+    repo: "*"
+    config:
+      tenant_id: 'private'
+  - cluster: '*'
+    repo: 'private'
+    config:
+      tenant_id: 'private'
+
+```
+This configuration is used both to apply tenantIDs to prowjobs, but is also used by Deck to filter out Tide information from orgRepos with tenantIDs that do not match. So even if you can lable all your prowjobs using cluster, make sure that all of your repos are given a tenantID as well.
+
+Once the PR is created, Prow operators should review the PR to make sure that no other TenantIDs were affected by the change.
```

Comment:
Should `Prow` be lower case?

Reply:
Prow seems to be uppercase elsewhere in the docs. Makes sense since it is the name of something.